Apromore-Frontend/src/loganimation/tokenAnimation.js

```diff
@@ -66,8 +66,11 @@ export default class TokenAnimation {
         this._TOKEN_LOG_GAP = 3;
```

Comment:
Consider making some constants static of the class, e.g. TokenAnimation.TOKEN_LOG_GAP

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static